src/java.base/share/conf/security/java.security

```diff
@@ -135,6 +135,26 @@ RestrictedSecurity1.javax.net.ssl.keyStore = NONE
 
 RestrictedSecurity1.securerandom.provider = SunPKCS11-NSS-FIPS
 RestrictedSecurity1.securerandom.algorithm = PKCS11
+
+RestrictedSecurity2.desc.name = CRIU
+RestrictedSecurity2.desc.number = 1
+RestrictedSecurity2.desc.policy = Security
+RestrictedSecurity2.desc.sunsetDate = 2030-01-01
```

Comment:
Is there a reason for this specific `desc.sunsetDate`?

Reply:
No, I was just about to add a comment about this. These first 4 descriptive properties can be changed to values that make sense.